src/work-allocation-2/containers/all-work-task/all-work-task.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from '@angular/core';
-import { Location } from 'api/workAllocation2/interfaces/task';
-import { Caseworker } from 'api/workAllocation2/interfaces/task';
+import { Location } from 'api/workAllocation2/interfaces/common';
+import { Caseworker } from 'api/workAllocation2/interfaces/common';
 import { UserInfo } from 'src/app/models/user-details.model';
```

Comment:
And this one is coming from `src/`, rather than a relative path.

Reply:
Actually, the two below as well...

Reply:
Probably going to get into trouble if I make changes to non 'my-case' related files, however I've updated the references above.